### DISCUSSION TITLE
Package deprecated tag in search list

### DISCUSSION THREAD

**Discussion by iamrekas** [npm/rfcs#513] [RRFC]
Hello,

I want suggest to improve searching. Add in search list new tag "the package is deprecated".

example:
![image](https://user-images.githubusercontent.com/17004593/150506866-01f3a664-daa9-48e8-a801-a88a70477549.png)

and inside the first one package:
![image](https://user-images.githubusercontent.com/17004593/150506989-f7b229a1-be03-42e9-9258-c7d155a9459b.png)

It probably will increase searching experience.

And other think, that you can also include the package type (like ts, dark ant etc.) or/and it's "esm" or "no-esm". because it's really be hellpfull.

**Comment by ljharb** [comment]
Every CJS package works with ESM, so there’d be very few packages with “no ESM” and virtually all of them with “ESM”.

Surfacing deprecation in search results, in particular, seems very useful.
